Assets/Scripts/Layers/LayerManager.cs

```diff
@@ -236,5 +242,13 @@ public bool IsDragOnButton()
         {
             return DragGhost && MouseIsOverButton;
         }
+
+        public void DeleteSelectedLayers()
+        {
+            foreach (var layer in LayerData.SelectedLayers)
```

Comment:
Kan dit ook niet beter in de LayerData class zelf te zitten? Dan hoef je de interne state van LayerData niet te exposen hiernaartoe en heb je sterkere encapsulatie

Reply:
LayerData is nu in principe een data object zonder functionaliteit, ik had eerst de variabelen van LayerData in de layerManager zelf, maar in overleg met jullie heb ik hier toen een apart object van gemaakt. 

Ik denk dat dit hetzelfde is als bovenstaande comment, ik kan LayerData meer uitbreiden, maar ik weet niet of dat helemaal de bedoeling was van die class